internal/test/overlay.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2018 Storj Labs, Inc.
```

Comment:
is this file a copy of pkg/overlay/mockoverlay.go? if so, any reason we can't just use that?

Reply:
It was a cyclical dependency issue. If you are alright with it, I can move `pkg/overlay/mockoverlay.go` completely to `internal/test/overlay.go`

Reply:
ok I removed `pkg/overlay/mockoverlay.go` and `internal/test/overlay.go` and made it `internal/mock/overlay.go`